### DISCUSSION TITLE
Deploying admin panel on another server documentation breaks reset password links

### DISCUSSION THREAD

**Discussion by micahbule** [strapi/strapi#8583] [Legacy Topics]
[Documentation in question](https://strapi.io/documentation/v3.x/admin-panel/deploy.html#deploy-the-administration-panel-on-another-server-aws-s3-azure-etc-than-the-api)

Consider the following configuration:
```
module.exports = ({ env }) => ({
  host: env('HOST', '0.0.0.0'),
  port: env.int('PORT', 1337),
  url: 'http://yourbackend.com',
  admin: {
    url: '/', // Note: The administration will be accessible from the root of the domain (ex: http://yourfrontend.com/)
    serveAdminPanel: false, // http://yourbackend.com will not serve any static admin files
  },
});
```

Configuring the `admin.url` property to `/` indeed serves your admin panel on the index route of your front-end domain. However with this config, password reset links being generated and sent out inherits the server URL (`https://yourbackend.com//auth/reset-password?code=${resetPasswordToken}` -- also note the extra slash after the domain) instead of the admin panel URL.

One workaround for this is to set the `admin.url` property to an absolute URL just like the server URL. Doing so would make the configuration look like:
```
module.exports = ({ env }) => ({
  host: env('HOST', '0.0.0.0'),
  port: env.int('PORT', 1337),
  url: 'http://yourbackend.com',
  admin: {
    url: 'http://yourfrontend.com',
    serveAdminPanel: false,
  },
});
```

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
